index.html

```diff
@@ -2,274 +2,295 @@
-                    <input type="text" name="lastName" placeholder="Last Name" maxlength="30" required />
-                </div>
+      <div class="item">
+        <h3>Profesional Art Printing Data</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia, molestiae quas vel sint commodi
+          repudiandae consequuntur voluptatum laborum numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga
+          praesentium optio, eaque rerum!</p>
 
-                <input type="email" name="email" placeholder="Email Adress" id="contactEmail" required />
+        <ul class="apply-flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
 
-                <textarea name="message" required maxlength="500">
-                    Enter text here
-                </textarea>
+        <button type="submit" class="btn-recent-works 4">See Project</button>
+      </div>
 
-                <p id="emailError" class="emailErrMessage">
-                </p>
+      <div class="item">
+        <h3>Profesional Art Printing Data</h3>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime mollitia, molestiae quas vel sint commodi
+          repudiandae consequuntur voluptatum laborum numquam blanditiis harum quisquam eius sed odit fugiat iusto fuga
+          praesentium optio, eaque rerum!</p>
 
-                <button type="submit" class="project-btn">Get in touch</button>
+        <ul class="apply-flex">
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="submit" class="btn-recent-works 5">See Project</button>
+      </div>
+    </section> <!-- end work section -->
 
-            </form>
-        </section>
 
-        <section class="footer-section-end flex">
+    <!-- start about me section -->
+    <section class="about-section" id="about">
+      <div class="about-me-section">
+        <h2>About me</h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <button>Get My Resume</button>
 
-            <ul class="flex">
-                <li><a href="">
-                        <a href="#"><img src="images/github.png" alt="git-icon" /></a>
-                    </a></li>
-                <li><a href="">
+        <div class="border-about-me-section"></div>
+      </div>
 
-                        <a href="#"><img src="images/linkedin.png" alt="linkden-icon" /></a>
-                    </a></li>
-                <li><a href="">
+      <div class="programming-languages apply-flex">
+        <div class="language">
+          <h4 class="languages">Language</h4>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
```

Comment:
- [ ] please kindly align the language frameworks and skills like that of figma.
yours
![Screenshot (654)_LI](https://user-images.githubusercontent.com/65298048/206567759-b7b94188-6d77-4440-884e-20030171d219.jpg)
figma design
![Screenshot (655)](https://user-images.githubusercontent.com/65298048/206567864-8e89e733-3a8f-4c50-9ceb-e4f3c09cae96.png)